app/bundles/CampaignBundle/Event/EventPreview.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
```

Comment:
Every new class should declare strict types.
```suggestion

declare(strict_types=1);

```